SerialPrograms/Source/Integrations/DiscordWebHook.cpp

```diff
@@ -16,7 +16,7 @@
 #include "Common/Cpp/PrettyPrint.h"
 #include "CommonFramework/GlobalSettingsPanel.h"
 #include "CommonFramework/Tools/Logger.h"
-#include "DiscordWebhook.h"
```

Comment:
I'm actually changing the everything to use the lowercase "h" since it's consistent with how Discord capitalizes it. lol

Reply:
Oops! Let me change them back.